libs/ui-lib/lib/common/components/clusterWizard/clusterDetailsValidation.ts

```diff
@@ -75,6 +75,7 @@ export const getClusterDetailsInitialValues = ({
     diskEncryptionTangServers: parseTangServers(cluster?.diskEncryption?.tangServers),
     diskEncryption: cluster?.diskEncryption ?? {},
     cpuArchitecture: cluster?.cpuArchitecture || getDefaultCpuArchitecture(),
+    externalPartnerIntegrations: cluster?.platform?.type === 'oci',
```

Comment:
Why is the externalPartnerIntegrations field a boolean here if it can take many values in practice, like nutanix/vsphere/oci/none?